README.md

```diff
@@ -1,6 +1,6 @@
-[![Build Status](https://img.shields.io/travis/kevinkjt2000/discord-game-server-status/develop.svg?style=flat-square&label=Travis-CI)](https://travis-ci.org/kevinkjt2000/discord-game-server-status?branch=develop)
-[![Coverage Status](https://img.shields.io/coveralls/kevinkjt2000/discord-game-server-status/develop.svg?style=flat-square&label=Coveralls)](https://coveralls.io/github/kevinkjt2000/discord-game-server-status?branch=develop)
-# Game Server Status Bot for Discord
+[![Build Status](https://img.shields.io/travis/kevinkjt2000//develop.svg?style=flat-square&label=Travis-CI)](https://travis-ci.org/kevinkjt2000/bowser?branch=develop)
+[![Coverage Status](https://img.shields.io/coveralls/kevinkjt2000//develop.svg?style=flat-square&label=Coveralls)](https://coveralls.io/github/kevinkjt2000/bowser?branch=develop)
```

Comment:
These links don't preview correctly for me

Reply:
I blame the awesome vim search/replace joys.

Reply:
Also, future change won't work anyway since the repo has not been renamed.